boards/arm/nrf52840_mdk/nrf52840_mdk_defconfig

```diff
@@ -0,0 +1,25 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF=y
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_NRF52840_MDK=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+CONFIG_ARM_MPU_NRF52X=y
```

Comment:
Here I think you can simply remove line 9

Reply:
seems to work